tailwind.config.cjs

```diff
@@ -1,8 +1,38 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  mode: "jit",
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
+  purge: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: "#84B0BA",
+        secondary: "#AE4646",
+        tertiary: "#822e2e",
+        white: "#FFFFFF",
+      },
+      fontSize: {
+        3.2: "3.2rem",
+        1.2: "1.2rem",
+      },
```

Comment:
If we will define some custom styles, we should name them. Random values will create a random design. We should rather strive towards making to shared elements components.

Could we use the built-in font sizes in tailwind? I believe the should be enough - https://tailwindcss.com/docs/font-size